src/channel.ts

```diff
@@ -1620,12 +1620,26 @@ export class Channel {
    * @return {Promise<CreateDraftResponse>} Response containing the created draft
    */
   async createDraft(message: DraftMessagePayload) {
-    return await this.getClient().post<CreateDraftResponse>(
+    const response = await this.getClient().post<CreateDraftResponse>(
       this._channelURL() + '/draft',
       {
         message,
       },
     );
+
+    if (response.draft) {
+      this.getClient().offlineDb?.executeQuerySafely(
```

Comment:
This query execution is missing tests.

Reply:
Decided to remove this and rely on events completely. Discussed offline.